VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.module.notification;
+
+import com.parse.ParsePushBroadcastReceiver;
+
+import org.edx.mobile.view.CourseDetailTabActivity;
+import org.edx.mobile.view.MyCoursesListActivity;
+
+/**
+ * subclass ParsePushBroadcastReceiver to provide fine control of
+ * app's behavior on receiving the notification message
+ */
+public class EdxParsePushBroadcastReceiver extends ParsePushBroadcastReceiver {
+
+    public void onReceive(android.content.Context context, android.content.Intent intent) {
+        super.onReceive(context,intent);
+     }
+
+    protected void onPushReceive(android.content.Context context, android.content.Intent intent) {
+        super.onPushReceive(context, intent);
+    }
+
+
+    protected void onPushDismiss(android.content.Context context, android.content.Intent intent) {
+        super.onPushDismiss(context, intent);
+    }
+
+    protected void onPushOpen(android.content.Context context, android.content.Intent intent) {
+        super.onPushOpen(context, intent);
+    }
+
+    protected java.lang.Class<? extends android.app.Activity> getActivity(android.content.Context context, android.content.Intent intent) {
+        return MyCoursesListActivity.class;
```

Comment:
This is the activity that will get shown when we receive a push? I assume we'll need to have more interesting logic here eventually - like sending you to a particular course?